### PR TITLE
Move files_trashbin to webpack

### PR DESCRIPTION
**Pull request by rullzer**
Combine the 2 js files and the scss. This saves compiling the scss on
the go. And going via Nextcloud storage to obtain it.

So 3 requests to 1. And this 1 request can be totally handled by the webserver. No need for NC to do anything :smile:

### CODE REVIEW

File: apps/files_trashbin/js/files_trashbin.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
+//# sourceMappingURL=files_trashbin.js.map
```
Comment (MorrisJobke) [inline apps/files_trashbin/js/files_trashbin.js]: Don't forget to add those files to the `.l10nignore`. cc @ChristophWurst as well.

File: apps/files_trashbin/.l10nignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#webpack bundled files
+js/files_trashbin.js
+js/files_trashbin.js.map
```
Comment (MorrisJobke) [inline apps/files_trashbin/.l10nignore]: `.map` files aren't scanned anyways ;)

**Comment by ChristophWurst** [comment]
So … do we want webpack configs for all the apps in this repo or will we just add entry points to the core webpack config? :thinking:

**Comment by rullzer** [comment]
> So … do we want webpack configs for all the apps in this repo or will we just add entry points to the core webpack config? thinking

Core webpack would be better.

File: apps/files_trashbin/src/webpack.dev.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  mode: 'development',
+  devServer: {
```
Comment (skjnldsv) [inline apps/files_trashbin/src/webpack.dev.js]: unneeded, we don't use the devserver

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: apps/files_trashbin/src/webpack.dev.js (hunk 3 above)
Comment (rullzer) [reply apps/files_trashbin/src/webpack.dev.js]: I just copied that from somewhere I think :)

Comment (skjnldsv) [reply apps/files_trashbin/src/webpack.dev.js]: Yes, it's all over the place X)
But this is a reliquat from the default webpack config we don't need here :(

**Comment by rullzer** [comment]
So moved to the single webpack config...

File: apps/files_trashbin/src/webpack.dev.js (hunk 3 above)
Comment (rullzer) [reply apps/files_trashbin/src/webpack.dev.js]: Resolved by moving to the main webpack config :wink:

**Review by ChristophWurst — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Comment by juliushaertl** [comment]
Failure seems unrelated https://drone.nextcloud.com/nextcloud/server/15780/260

**Pull request merged**